app/src/main/java/com/globant/franciscoprieto/kotlincalculator/mvp/presenter/CalculatorPresenter.kt

```diff
@@ -1,36 +1,82 @@
 package com.globant.franciscoprieto.kotlincalculator.mvp.presenter
```

Comment:
the lack of unit tests is disturbing, please add.

Reply:
done!